### ISSUE TITLE
[Metabase Version 0.45.1] Bigquery doesn't display the summarize column

### ISSUE THREAD

**Issue by duongnt18** [metabase/metabase#27437]
**Describe the bug**
With Bigquery Database, the summarize column doesn't display when we use the editor
<img width="1003" alt="image" src="https://user-images.githubusercontent.com/19161026/209759809-923b9ffe-0954-41f8-b14f-47cf682a4bce.png">

**To Reproduce**
1. Go to the table on Bigquery connection
2. Let's summarize some calculation field
3. Let's summarize again on these above summarize field

Information about your Metabase Installation:

Metabase version: 0.45.1
Metabase hosting environment: Docker
Metabase internal database: Mariadb

**Comment by flamber** [comment]
Hi @duongnt18 
I doubt it has anything to do with BigQuery, and I cannot reproduce, but it's causing the same problem as #24839 so closing in favor of that. Try clicking Visualize after step 2 (perhaps do a browser refresh), then step 3 should work. Seems like it's a race-condition that can occur sometimes.

**Comment by duongnt18** [comment]
Hi @flamber , it's normal in the Sample Database.
This error only occur when i update to 0.45.1. 
Please check my our record screen 

https://user-images.githubusercontent.com/19161026/209794452-09b020bc-f3f2-4ca5-be91-a4505c51a3c0.mp4

Thanks
Duong Nguyen

**Comment by flamber** [comment]
@duongnt18 What you are showing there is the same problem as #12248.

**Comment by duongnt18** [comment]
Hi @flamber , first i am sorry if my explain is confuse.
Please take a look at the record below, the left hand side is 0.45.1, and the right hand side is 0.44.6.
The second summarize on 0.45.1 doesn't show, while i can do it in all previous version:

https://user-images.githubusercontent.com/19161026/209796992-64697b89-41ae-42fd-87d8-1a6ac029e123.mp4

Thanks
Duong Nguyen

**Comment by flamber** [comment]
I've been able to reproduce on one of my instances, but not on any other instance. It has nothing to do with BigQuery, but seems to be related to general table metadata and causes other problems as well. Let me work on this some more...

**Comment by duongnt18** [comment]
Great ^^. 
Thanks @flamber

**Comment by flamber** [comment]
@duongnt18 I've created #27462 - upvote by clicking :+1: on the first post
